<launch>
    <arg name="folder_path" default="jaguaruco"/>
    <arg name="todo" default="aruco"/>
    <arg name="calibration_file" default="CalibData.json"/>
    <arg name="frequency" default="1" />
    <arg name="img_format" default=".jpg" />
    <arg name="aruco_dictionary" default="DICT_4X4_200" />
    <arg name="aruco_size" default="0.05" />
    <!--
    <node name="elstart" pkg="scopro" type="elstart.py" output="screen">
        <param name="calibration_file" value="$(arg calibration_file)" />
        <param name="dataset" value="$(arg dataset_path)" />
        <param name="dataset" value="$(arg dataset_path)" />
    </node>
    -->
    <node name="image_publisher" pkg="scopro" type="Convertion.py" output="screen">
        <param name="frequency" value="$(arg frequency)" />
        <param name="img_format" value="$(arg img_format)" />
        <param name="folder" value="$(arg folder_path)" />
        <param name="todo" value="$(arg todo)" />
    </node>

    <node name="correct_images" pkg="scopro" type="Correction.py" output="screen">
        <param name="calibration_file" value="$(arg calibration_file)" />
    </node>

    <node name="aruco_detector" pkg="scopro" type="aruco_pose.py" output="screen">
        <param name="aruco_dictionary" value="$(arg aruco_dictionary)" />
        <param name="aruco_size" value="$(arg aruco_size)" />
        <param name="calibration_file" value="$(arg calibration_file)" />
    </node>

</launch>
